docs/source/posts/2022/2023-02-01-final-report-shivam.rst

```diff
@@ -0,0 +1,406 @@
+   information to a ``.gltf`` or ``.glb`` file format.
+
+   *Pull Requests:*
+
+   -  **Exporting scene as glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+   -  **Exporting scene as glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/630
+
+
+Simple Actor Animations
+***********************
+
+   Added simple actor animations (translation, rotation & scale of
+   actors) support. The animation data (transformation and timestamp) is
+   stored in buffers. It converts the binary data to ndarrays and
+   creates a timleline for each animation in glTF animations. This
+   timeline containsactors an can be added to the scene. We can animate
+   the scene by updating timline inside a timer callback.
+
+   https://user-images.githubusercontent.com/74976752/194863125-58f3717d-d89e-48e7-8e2c-a8501e4f230b.mp4
+
+   *Pull Requests:*
+
+   -  **Simple Animations in glTF: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+   -  **Simple Animations in glTF Tutorial: (Merged)**
+      https://github.com/fury-gl/fury/pull/643
+
+
+Morphing in glTF
+****************
+
+   glTF allows us to animate meshes using morph targets. A morph target
+   stores displacements or differences for certain mesh attributes. At
+   runtime, these differences may be added to the original mesh, with
+   different weights, to animate parts of the mesh. Added methods to
+   extract this information, update the timeline and apply morphing to
+   each actor in the scene.
+
+   https://user-images.githubusercontent.com/74976752/194862679-ce239e11-5373-4fc5-95a7-4be12feb99cb.mp4
+
+   *Pull Requests:*
+
+   -  **Morphing support in glTF: (Under Review)**
+      https://github.com/fury-gl/fury/pull/700
+   -  **Morphing in glTF demo: (Under Review)**
+      https://github.com/fury-gl/fury/pull/700
+
+
+Skeletal Animations (Skining)
+*****************************
+
+   Another way of animating a glTF is by skinning. It allows the
+   geometry (vertices) of a mesh to be deformed based on the pose of a
+   skeleton. This is essential in order to give animated geometry. It
+   combines every parameter of a glTF file. While working with skinning,
+   we need to keep track of the parent-child hierarchy of
+   transformations. Vertex Skinning takes full advantage of newly
+   implemented ``Timeline`` & ``Animation`` modules to track
+   hierarchical transformation order. Though the current version of the
+   skinning implementation works with most of the glTF sample modes, It
+   struggles with models that have multiple actors (e.g. BrainStem). It
+   can be fixed by using the vertex shader to update the vertices. The
+   current implementation of skinning supports the following:
+
+   -  Multiple animation support
+   -  Multiple node and multiple actor animation with textures
+   -  Show or hide bones/skeleton of the model.
+
+   https://user-images.githubusercontent.com/74976752/194862048-7ce65b42-2717-436b-b311-85368f3c3714.mp4
```

Comment:
vidéos integrations does not work:

![image](https://user-images.githubusercontent.com/23106443/216668706-dce346ba-a642-415e-9b0a-e5aa2baa13e4.png)